flask_mapper/flask_mapper/sheets.py

```diff
@@ -0,0 +1,105 @@
+import httplib2
+import os
+import googlemaps
+from apiclient import discovery
+from oauth2client import client
+from oauth2client import tools
+from oauth2client.file import Storage
+
+from local_settings import GOOGLE_MAPS_KEY
+
+gmaps = googlemaps.Client(key=GOOGLE_MAPS_KEY)
+
+# If modifying these scopes, delete your previously saved credentials
+# saved in credentials.json
+SCOPES = 'https://www.googleapis.com/auth/spreadsheets.readonly'
+CLIENT_SECRET_FILE = os.path.join(os.getcwd(),
+                                  'flask_mapper/flask_mapper'
+                                  '/client_secret.json')
+APPLICATION_NAME = 'Flask Mapper'
+CREDENTIALS = os.path.join(os.getcwd(),
+                           'flask_mapper/flask_mapper/credentials.json')
+
+if __name__ == "__main__":
+    try:
+        import argparse
+        flags = argparse.ArgumentParser(parents=[tools.argparser]).parse_args()
+    except ImportError:
+        flags = None
+
+
+def get_credentials():
+    """
+    Gets valid user credentials from storage.
+
+    If nothing has been stored, or if the stored credentials are invalid,
+    the OAuth2 flow is completed to obtain the new credentials.
+
+    Returns:
+    Credentials, the obtained credential.
+    """
+    store = Storage(CREDENTIALS)
+    credentials = store.get()
+    if not credentials or credentials.invalid:
+        print("Credentials: %s" % credentials)
```

Comment:
Remove prints before checkin

Reply:
Got it, before this goes further I'll add tests and linting to catch these.